pkg/cluster/base.go

```diff
@@ -1314,3 +1315,16 @@ func (p *ProviderBase) ValdiateRequireSSHPrivateKey() error {
 	}
 	return nil
 }
+
+func GetDefaultTemplate(provider string) ([]byte, error) {
+	templateBytes := []byte(common.DefaultTemplates)
+	templates := map[string]interface{}{}
+	err := yaml.Unmarshal(templateBytes, &templates)
```

Comment:
Maybe you can move it into `init()` function and don't need to unmarshal the content everytime.